02-GitHub.Rmd

```diff
@@ -9,85 +9,94 @@ opts_chunk$set(tidy.opts=list(width.cutoff=50), tidy = FALSE)
 
 ```{r image1, echo=FALSE, out.width="700px", fig.align='center'}
 knitr::include_graphics("images/Git0.PNG")
 ```
 
 Image credit: devmountain.com
 
-## .git the language {#Git2.3}
+## The Language of Git {#Git2.3}
 
-Were you the kid that was tormented by your siblings speaking Pig Latin (also known as "Igpay Atinlay") and no matter how hard you tried, you couldn't figure it out. Learning to use .git and Github can feel like learning a new language. But we promise it is easier than Pig Latin.
+Were you the kid that was tormented by your siblings speaking Pig Latin (also known as "Igpay Atinlay") and no matter how hard you tried, you couldn't figure it out? Learning to use Git and Github can feel like learning a new language. But we promise it is easier than Pig Latin.
 
-When working with .git and GitHub, you are going to see the words `repo`, `origin`, `commit`, `push`, `pull`, `clone`, `fork`, and `branch`, among others. We will start with some basic definitions here, and in later chapters will elaborate on terms as necessary.
+When working with Git and GitHub, you are going to see the words `repo`, `origin`, `commit`, `push`, `pull`, `clone`, `fork`, and `branch`, among others. We will start with some basic definitions here, and in later chapters will elaborate on terms as necessary.
 
 **repo(sitory)** - a folder whose contents are is tracked by Git. A repo can exist locally (on your computer) as a remote repository (on GitHub), or both.
 
-**origin** - the primary .git repo, against which other copies are compared. In our examples, we will create the origin repo in GitHub
+**origin** - the primary Git repo, against which other copies are compared. In our examples, we will create the origin repo in GitHub
 
-**push** - sends changes from your local .git repo to the remote repository (e.g. GitHub)
+**push** - send changes from your local Git repo to the remote repository (e.g. GitHub)
 
-**pull** - gets changes from the cloud and updates your local .git repo
+**pull** - get changes from the cloud and updates your local Git repo
 
 > Note: this is different from a `pull request` which is when you want someone else to incorporate your changes into their GitHub repo. More on this later.
 
-**commit** - create a named version of a set of one or more changes to a local .git repo, to be pushed to the cloud. Also used to refer to this named state. In other words, when you commit your changes, you create a commit.
+**commit** - create a named version of a set of one or more changes to a local Git repo. 
+Like a bookmark of the current changes made. Often will be pushed to GitHub, but can be local only.
+Can be a verb and a noun!
+In other words, when you commit your changes, you create a commit.
 
-**clone** - creates a copy of a git repo. In our examples you the repos will first be created in GitHub (i.e., origin) and then cloned to your local .git repo on your computer.
```

Comment:
I had to look this up, but I think the clone is actually the most closely linked copy. If you clone a repo you have access to, you can make direct changes to it. The only thing is that if you don't have access to the repo, it's effectively unlinked. 

Also, perhaps not important at this stage, but you can always make a fork a clone and viseversa, just by changing where the remotes and upstreams point to.